jenkins-plugin/src/test/java/org/jenkinsci/plugins/pipeline/maven/AbstractIntegrationTest.java

```diff
@@ -71,12 +81,31 @@ public abstract class AbstractIntegrationTest {
     public static void setupWatcher() {
         buildWatcher = new BuildWatcher();
         runBeforeMethod(buildWatcher);
+
+    }
+
+    public static void initializeH2(Jenkins jenkins) throws Exception {
+        File databaseRootDir = new File("target", "h2-test");
+        String jdbcUrl = "jdbc:h2:file:" + new File(databaseRootDir, "jenkins-jobs").getAbsolutePath() + ";" +
+                "AUTO_SERVER=TRUE;MULTI_THREADED=1;QUERY_CACHE_SIZE=25;JMX=TRUE";
+        GlobalPipelineMavenConfig.get().setJdbcUrl(jdbcUrl);
+
+        String credsId = "jdbcCredsId";
+        UsernamePasswordCredentialsImpl c =
+                new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL, credsId, "sample", "sa", "sa");
+        CredentialsProvider.lookupStores(jenkins).iterator().next().addCredentials(Domain.global(), c);
+        GlobalPipelineMavenConfig.get().setJdbcCredentialsId(credsId);
```

Comment:
this is not safe to be called in a static - as you are calling from a method (setup) it makes this safe - but this method shoud not be static.